accounting/service.proto

```diff
@@ -16,6 +16,11 @@ import "session/types.proto";
 // accounts are possible, if both use the same token type.
 service AccountingService {
   // Returns the amount of funds in GAS token for the requested NeoFS account.
+  //
+  // Statuses:
+  // - **OK** (0, [SECTION_SUCCESS](status.md#neo.fs.v2.status.Success)):
```

Comment:
GitHub markdown cross referencing must not be used in API spec.

Reply:
Removed.